ieeeuoft/makeuoft-site-2021
# Styled all the pages that the user will see

## Steps to QA

- Create a superuser account
- Just scroll, click, resize the page as much as you can to see if things look ugly
